sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -128,6 +129,17 @@ def test_n_neighbors_datatype():
                         neighbors_.kneighbors, X=X, n_neighbors=3.)
 
 
+def test_not_fitted_error_gets_raised():
+    expected_msg = "This NearestNeighbors instance is " \
+                   "not fitted yet. Call 'fit' with appropriate" \
+                   " arguments before using this method"
+    neighbors_ = neighbors.NearestNeighbors()
+    assert_raises_regex(NotFittedError, expected_msg,
```

Comment:
I think you can simply use `assert_raise_message` here

Reply:
Thanks. Will change.